### PR TITLE
updated Project section

### PR DESCRIPTION
**Pull request by kanikakaur27**

### CODE REVIEW

File: src/assets/stylesheets/project.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    width: 350px;
+    height: 350px;
+}
+.card-h2{
+    font-size: 50px;
+    margin-left: 10px;
+    font-weight: boldest;
+}
+.card-text{
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```
Comment (Gauravsharma-20) [inline src/assets/stylesheets/project.css:13]: centralize font-family in the main index.css file and remove from here.

File: src/assets/stylesheets/project.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    margin-left: 10px;
+}
+.card-h4{
+    font-size: 20px;
+    margin-left: 10px;
+}
+
+.card-link{
+    padding-left: 6px;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```
Comment (Gauravsharma-20) [inline src/assets/stylesheets/project.css:25]: this too

File: src/assets/stylesheets/project.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+}
+.card-desc{
+    padding: 30px;
+}
+.project-tile{
+    background-color: lightblue;
+    display: inline-flex;
+    margin: 30px 50px 30px 50px;
+    align-items: center;
+    z-index: 10;
```
Comment (Gauravsharma-20) [inline src/assets/stylesheets/project.css:38]: any reason for using z-index here?

**Review by Gauravsharma-20 — CHANGES_REQUESTED** [review]

**Comment by HarshDutt17** [comment]
merging the pr for now, as project.css is required, for other files, but do the requested changes soon on a separate pr.

**Pull request merged**